src/Microsoft.ML.Transforms/UngroupTransform.cs

```diff
@@ -232,44 +232,75 @@ public PivotColumnInfo(string name, int index, int size, PrimitiveType itemType)
             }
 
             private readonly Schema _inputSchema;
+            public int InputColumnCount => _inputSchema.Count;
```

Comment:
>public int InputColumnCount => _inputSchema.Count; [](start = 12, length = 50)

can you move it next to OutputSchema? that layout of public property surrounded of private readonly doesn't looks good.